runtime.md

```diff
@@ -15,9 +15,9 @@ This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
 * **`status`**: (string) is the runtime state of the container.
 The value MAY be one of:
-    * `created`: the container has been created but the user-specified code has not yet been executed
-    * `running`: the container has been created and the user-specified code is running
-    * `stopped`: the container has been created and the user-specified code has been executed but is no longer running
+    * `created`: the container process has neither exited nor executed the user-specified code
+    * `running`: the container process has executed the user-specified code but has not exited
+    * `stopped`: the container process has exited
```

Comment:
It would be better to state that in the stopped state the container has ran the container's main process and it has exited leaving the container in the stopped state